specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json

```diff
@@ -0,0 +1,366 @@
+        ],
+        "description": "Security contact configurations for the subscription",
+        "operationId": "SecurityContacts_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SecurityContactList": {
+      "type": "object",
+      "description": "List of security contacts response",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "description": "List of security contacts",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityContact"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "SecurityContact": {
+      "type": "object",
+      "description": "Contact details for security issues",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Security contact data",
+          "$ref": "#/definitions/SecurityContactProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "SecurityContactProperties": {
+      "type": "object",
+      "description": "Describes security contact properties",
+      "properties": {
+        "emails": {
+          "type": "string",
+          "description": "The security contact's emails list as a string. Emails will be separated by ';'"
+        },
+        "phone": {
+          "type": "string",
+          "description": "The security contact's phone number"
+        },
+        "alertNotifications": {
+			"type" : "object",
+			"description": "Defines whether to send security alerts notifications to the security contact and for what severities",
+			"properties": {
+				"state" : {
+					"type": "string",
+					"description" : "Defines if the alertNotifications is enabled",
+          			"enum": [
+            			"On",
+            			"Off"
+          			],
+          			"x-ms-enum": {
+            		"name": "state",
+            		"modelAsString": true,
+            		"values": [
+						{
+							"value": "On",
+							"description": "Get notifications on new alerts"
+						},
+						{
+							"value": "Off",
+							"description": "Don't get notifications on new alerts"
+						}
+            		]
+					}
+				},
```

Comment:
consider defining the "state" property as a separate definition, as it is used here (alertNotifications) and by notificationsByRole

Reply:
As we spoke this is going to be left as is